api/Dockerfile

```diff
@@ -65,6 +65,10 @@ RUN apt-get update \
       && rm -rf /var/lib/apt/lists/* \
     && mkdir -p /var/log/openverse_api/openverse_api.log
 
+# Copy the operational code into the final image
+COPY run.sh manage.py /api/
+COPY catalog/ /api/catalog/
```

Comment:
Could we write 
```dockerfile
COPY . /api
```
like we did for analytics?

Reply:
^ This would make the `COPY` statement
- same in all Dockerfiles
- identical to the `volumes` rules from the Docker-Compose file

Reply:
My only concern with doing this is that it copies _all_ files in each directory, which includes a number of files which are irrelevant for production use (including all of the tests). I've generally been steered away from that, but it might not be _too_ much extra space. What do you think?

Reply:
Would a `.dockerignore` file help with that? We could reduce a lot of cruft from going into the context.

Reply:
I also submitted #442 which would make the `analytics/` code structure similar to `ingestion_server/`. If you could maybe rebase this PR on top of that one, the same exclusion strategy would work for both codebases.

Reply:
Can do! This is also a good reminder to review that PR :smile: